src/sharing/components/ShareAutosuggest.jsx

```diff
@@ -104,19 +117,22 @@ export default class ShareAutocomplete extends Component {
     setTimeout(() => this.input.focus(), 1)
   }
 
-  renderInput(props) {
+  renderInput(inputProps) {
+    const { loading, recipients } = this.props
     return (
       <div className={styles['recipientsContainer']}>
-        {this.props.recipients.map((recipient, i) => {
+        {recipients.map((recipient, idx) => {
           const recipientEmail = recipient.id
             ? getPrimaryEmail(recipient)
             : recipient.email
+
+          const value = recipientEmail || recipient.fullname
```

Comment:
We have somewhere else also the same logic : if no email then take fullname (https://github.com/cozy/cozy-drive/pull/1604/files#diff-3e73a3435de20ef054b2ae21b9b58a37R35) . Can we have one place to do the logic? It's error prone in the future if we want to change the behavior (